doc-content/enterprise-only/employee-rostering/er-deploy-prepare-proc.adoc

```diff
@@ -7,9 +7,36 @@ You must download and prepare the deployment files before building and deploying
 
 . Download the `{PRODUCT_FILE}-add-ons.zip` file from the {PRODUCT_DOWNLOAD_LINK}[Software Downloads] page for {PRODUCT} {PRODUCT_VERSION}.
 . Unzip the downloaded archive.
-. Unzip the employee rostering zip file (`{PRODUCT_FILE}-employee-rostering.zip`) that is extracted from the add-ons archive.
-
-.Result
-When you unzip the employee rostering zip file, the `optashift-employee-rostering-{MAVEN_ARTIFACT_VERSION}` folder is created. This folder is the base folder in subsequent steps. 
-
-NOTE: File and folder names might have higher version numbers than specifically noted in this document.
+. Unzip the employee rostering zip file (`{PRODUCT_FILE}-employee-rostering.zip`) that is extracted from the add-ons archive. 
++
+The `optashift-employee-rostering-{MAVEN_ARTIFACT_VERSION}` folder is created. This folder is the base folder in subsequent steps. 
```

Comment:
I would not want to do this, because "subsequent steps" refer to multiple sections and listing them all would create a maintainability problem (what if a new section is added later?). Changes to "subsequent parts of this document".